From f5793b22eca3ad8111446c1a3722bb73af67a450 Mon Sep 17 00:00:00 2001
From: Davee Nguyen <davee@daveenguyen.com>
Date: Mon, 18 Mar 2019 23:51:03 -0700
Subject: [PATCH] Use implicit return None on generator functions

---
 pyscreeze/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyscreeze/__init__.py b/pyscreeze/__init__.py
index 544e88f..58d5b28 100644
--- a/pyscreeze/__init__.py
+++ b/pyscreeze/__init__.py
@@ -159,7 +159,7 @@ def _locateAll_opencv(needleImage, haystackImage, grayscale=None, limit=10000, r
         if USE_IMAGE_NOT_FOUND_EXCEPTION:
             raise ImageNotFoundException('Could not locate the image (highest confidence = %.3f)' % result.max())
         else:
-            return None
+            return
 
     # use a generator for API consistency:
     matchx = matches[1] * step + region[0]  # vectorized
@@ -258,7 +258,7 @@ def _locateAll_python(needleImage, haystackImage, grayscale=None, limit=None, re
         if USE_IMAGE_NOT_FOUND_EXCEPTION:
             raise ImageNotFoundException('Could not locate the image.')
         else:
-            return None
+            return
 
 
 def locate(needleImage, haystackImage, **kwargs):
